ext/B/B/Concise.pm

```diff
@@ -1489,6 +1489,10 @@ They're opcode specific, and occur less often than the public ones, so
 they're represented by short mnemonics instead of single-chars; see
 B::Op_private and F<regen/op_private> for more details.
 
+Note that a number after a '/' might refer to the number of child OPs
+if the OP has not been privatized by a check routine. (e.g. sassign
+has 2 child OPs in the example above.)
+
```

Comment:
I don't what it means for an op to be "privatized".

Reply:
Neither do I. That terminology is straight out of op.h:
````
 *	op_private	Flags peculiar to a particular operation (BUT,
 *			by default, set to the number of children until
 *			the operation is privatized by a check routine,
 *			which may or may not check number of children).
 ````

Reply:
That description of `op_private` goes back at least as far as [perl 5.0 alpha 2](https://github.com/Perl/perl5/commit/79072805bf63abe5b5978b5928ab00d360ea3e7f)